package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "cordova-sms-plugin",
-  "version": "1.0.1",
+  "version": "1.0.1-dev-1.0",
```

I'll bump the version in another commit.
```suggestion
  "version": "1.0.1",
```